install.sh

```diff
@@ -200,6 +204,38 @@ file_not_grpowned() {
   [[ " $(id -G "$USER") " != *" $(get_group "$1") "*  ]]
 }
 
+# Please sync with 'test-ruby()' in 'Library/Homebrew/utils/ruby.sh' from Homebrew/brew repository.
+test-ruby () {
+  [[ ! -x $1 ]] && { echo "false"; return 1; }
+  "$1" --enable-frozen-string-literal --disable=gems,did_you_mean,rubyopt -rrubygems -e \
+    "puts Gem::Version.new(RUBY_VERSION.to_s.dup).to_s.split('.').first(2) == \
+          Gem::Version.new('$required_ruby_version').to_s.split('.').first(2)" 2>/dev/null
+}
+
+usable_ruby() {
+  local ruby_exec
+  ruby_exec=$(which ruby)
+  test "$(test-ruby "$ruby_exec")" == true
+}
+
+usable_glibc() {
+  local glibc_version
+  glibc_version=$(ldd --version | head -n1 | grep -o '[0-9.]*$' | grep -o '^[0-9]\+\.[0-9]\+')
+  version_ge "$glibc_version" $required_glibc_version
```

Comment:
```suggestion
  version_ge "$glibc_version" "$required_glibc_version"
```
Quotes for consistency and style.

Reply:
Quotes around `required_glibc_version` are not necessary because we set its value ourselves, it does not have spaces, and this is not expected to change. 
What style do you mean?

Reply:
All `$foo` should be surrounded by quotes `"$foo"`. Each time I see `$foo` I have to ask myself, Is this a bug? 99% of the time the answer is yes.

Reply:
If I see `$required_glibc_version` I have to scroll up to the top of the file to find the definition of `required_glibc_version` to determine whether it's a bug. If I see `"$required_glibc_version"` I can happily continue on my way.